src/main/kotlin/com/autonomousapps/internal/advice/ConsoleReport.kt

```diff
@@ -23,37 +22,37 @@ internal data class ConsoleReport(
 
   companion object {
     fun from(computedAdvice: ComputedAdvice): ConsoleReport {
-      var addToApiAdvice = emptySet<Advice>()
-      var addToImplAdvice = emptySet<Advice>()
-      var removeAdvice = emptySet<Advice>()
-      var changeToApiAdvice = emptySet<Advice>()
-      var changeToImplAdvice = emptySet<Advice>()
-      var compileOnlyDependencies = emptySet<Dependency>()
-      var unusedProcsAdvice = emptySet<Advice>()
+      var addToApiAdvice = emptyList<Dependency>()
+      var addToImplAdvice = emptyList<Dependency>()
+      var removeAdvice = emptyList<Dependency>()
+      var changeToApiAdvice = emptyList<Dependency>()
+      var changeToImplAdvice = emptyList<Dependency>()
+      var compileOnlyDependencies = emptyList<Dependency>()
+      var unusedProcsAdvice = emptyList<Dependency>()
 
       if (!computedAdvice.filterRemove && computedAdvice.removeAdvice.isNotEmpty()) {
-        removeAdvice = computedAdvice.removeAdvice
+        removeAdvice = computedAdvice.removeAdvice.map { it.dependency }
```

Comment:
fyi there's a `mapToSet` extension function available to simplify creating sets.